api/src/routing.js

```diff
@@ -209,23 +209,13 @@ app.all(`/${db.getPath()}/export/:type/:form?`, exportData.routeV1);
 app.get('/api/v2/export/:type', exportData.routeV2);
 app.postJson('/api/v2/export/:type', exportData.routeV2);
 
-app.post('/api/v1/records', [jsonParser, formParser], function(req, res) {
-  auth.check(req, 'can_create_records', null, function(err) {
-    if (err) {
-      return serverUtils.error(err, req, res, true);
-    }
-    records.create(req, req.is(['json','urlencoded']))
-      .then(res.json)
-      .catch(err => {
-        serverUtils.serverError(err, req, res);
-      });
-  });
-});
+app.post('/api/v1/records', [jsonParser, formParser], records.v1);
+app.post('/api/v2/records', [jsonParser, formParser], records.v2);
 
 app.get('/api/v1/forms', function(req, res) {
   forms.listForms(req.headers, function(err, body, headers) {
     if (err) {
-      return serverUtils.serverError(err, req, res);
+      return serverUtils.error(err, req, res);
```

Comment:
did you mean to change this at the same time?

Reply:
Yeah, I noticed it was also wrong while I was in there, fixing the records api.